src/web/views/macros/htbhf-child-dob-input.njk

```diff
@@ -10,50 +10,55 @@
 
 {% macro htbhfChildDobInput(params) %}
 
-  {% call govukFieldset({
-      legend: {
-        text: params.text.legend,
-        classes: "govuk-fieldset__legend--m"
-      }
-   }) %}
+  <div class="c-htbhf-child-dob-input">
 
-    {{ govukInput({
-      label: {
-        text: params.nameLabel
-      },
-      id: 'childDob-name-' + params.index,
-      name: 'childDobName-' + params.index,
-      value: params.children['childDobName-'  + params.index],
-      errorMessage: params.errors | getErrorForField('childDobName-'  + params.index),
-      autocomplete: "off"
-    }) }}
+    {% call govukFieldset({
+        legend: {
+          text: params.text.legend,
+          classes: "govuk-fieldset__legend--m"
+        }
+     }) %}
 
-  {% endcall %}
-
-  {{ htbhfDateInput({
-    fieldNamePrefix: "childDob-" + params.index,
-    claim: params.children,
-    errors: params.errors,
-    hint: {
-      text: params.hint
-    },
-    legend: {
-      text: params.text.dateLegend,
-      classes: "govuk-fieldset__legend--m"
-    }
-  }) }}
-
-  {% if params.showRemoveButton %}
-    {{ govukButton({
-      text: params.text.removeButton,
-      classes: "govuk-button--secondary",
-      type: "submit",
-      value: params.index,
-      name: "remove",
-      attributes: {
-        id: "remove-child-" + params.index
+      {{ govukInput({
+        label: {
+          text: params.nameLabel
+        },
+        id: 'childDob-name-' + params.index,
+        name: 'childDobName-' + params.index,
+        value: params.children['childDobName-'  + params.index],
+        errorMessage: params.errors | getErrorForField('childDobName-'  + params.index),
+        autocomplete: "off"
+      }) }}
+
+    {% endcall %}
+
+    {{ htbhfDateInput({
+      fieldNamePrefix: "childDob-" + params.index,
+      claim: params.children,
+      errors: params.errors,
+      hint: {
+        text: params.hint
+      },
+      legend: {
+        text: params.text.dateLegend,
+        classes: "govuk-fieldset__legend--m"
       }
     }) }}
-  {% endif %}
 
+    {% if params.showRemoveButton %}
+      <div class="c-htbhf-child-dob-input__remove-button">
+        {{ govukButton({
+          text: params.text.removeButton,
+          classes: "govuk-button--secondary",
+          type: "submit",
+          value: params.index,
+          name: "remove",
+          attributes: {
+            id: "remove-child-" + params.index
+          }
+        }) }}
+      </div>
+    {% endif %}
+  </div>
+  <hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
```

Comment:
I think possibly the `hr` should be in the parent template. In this file it means every time the component is used it will have a line at the bottom. My thoughts are the line is required due to stacked / multiple child inputs which is a feature of the parent component. This component would be more reusable if it didn't assume this structure / style itself.

Currently, if a design decision was made to remove the `hr` from the last instance of the component in the list it would be tricky to deal with - however it should be trivial if it was in the parent component (which has knowledge of the render loop).

Reply:
done